components/BusinessList.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="mt-5 ml-3">
+    <!-- Header -->
+    <h2 class="headline-lg">
+      Einzigartige <b>{{ type }}</b> aus <b>{{ city }}</b>
+    </h2>
+    <div v-swiper="swiperOptions" class="overflow-visible">
+      <!-- Swipe Buttons -->
+      <div class="d-flex list-buttons">
+        <v-btn icon class="swiper-button-next">
+          <v-icon>mdi-chevron-right</v-icon>
+        </v-btn>
+        <v-btn icon class="swiper-button-prev">
+          <v-icon>mdi-chevron-left</v-icon>
+        </v-btn>
+      </div>
+      <!-- List -->
+      <div class="swiper-wrapper">
+        <business-card
+          v-for="business in getFirstBusinesses(numberOfBusinesses)"
+          :key="business.id"
+          :city="city"
+          :business="business"
+          class="swiper-slide"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { directive } from 'vue-awesome-swiper'
+import { mapGetters } from 'vuex'
+import BusinessCard from './BusinessCard.vue'
+
+export default {
+  name: 'BusinessList',
+
+  components: {
+    BusinessCard,
+  },
+
+  directives: {
+    swiper: directive,
+  },
+
+  data: () => ({
+    numberOfBusinesses: 5,
+    type: 'Arbeitgeber',
+    city: 'Passau',
+    swiperOptions: {
+      spaceBetween: 30,
+      slidesPerView: 'auto',
+      navigation: {
+        prevEl: '.swiper-button-prev',
```

Comment:
i think this could cause problems if you mount the component twice, because the classes do not have a scope.
So swiper two would probably also bind the buttons from swiper one.